client/state/comments/selectors.js

```diff
@@ -62,8 +59,9 @@ export const getPostMostRecentCommentDate = createSelector(
  */
 export const getPostOldestCommentDate = createSelector(
 	( state, siteId, postId ) => {
-		const items = getPostCommentItems( state, siteId, postId );
-		return items && items.last() ? new Date( items.last().get( 'date' ) ) : undefined;
+		const items = filter( getPostCommentItems( state, siteId, postId ), { parent: false } );
+		const oldestComment = minBy( items, item => new Date( item.date ) );
```

Comment:
I wasn't trusting the order on the state. Fixed on 18bc439ab4db1a1d243a4901d426fcb5f5fec705.